doc/classes/CameraEffects.xml

```diff
@@ -37,5 +37,20 @@
 		<member name="override_exposure_enabled" type="bool" setter="set_override_exposure_enabled" getter="is_override_exposure_enabled" default="false">
 			If [code]true[/code], overrides the manual or automatic exposure defined in the [Environment] with the value in [member override_exposure].
 		</member>
+		<member name="vignette_center" type="Vector2" setter="set_vignette_center" getter="get_vignette_center" default="Vector2(0.5, 0.5)">
+			The center to use for vignetting ([code]Vector2(0.5, 0.5)[/code] is the center of the viewport). Only used if [member vignette_intensity] is greater than [code]0.0[/code].
```

Comment:
Note that for VR/stereo rendering the center of the vignette will be different for each eye and can be calculated by running: 
```
vec4 proj_center = PROJECTION_MATRIX * vec4(0.0, 0.0, z_far, 1.0);
vec2 center = proj_center.xy / proj_center.w;
```
I do have a separate solution already in our Godot XR toolkit for vignettes already but if we want to support VR here, that will need to be added in.